addons/mail/static/src/discuss/call/common/call.xml

```diff
@@ -3,10 +3,11 @@
 
     <t t-name="discuss.Call">
         <PttAdBanner/>
-        <div class="o-discuss-Call user-select-none d-flex" t-att-class="{'o-fullscreen fixed-top vw-100 vh-100': state.isFullscreen, 'o-compact': props.compact,'o-minimized': minimized, 'position-relative': !state.isFullscreen }">
-            <div class="o-discuss-Call-main d-flex flex-grow-1 flex-column align-items-center justify-content-center position-relative overflow-auto" t-on-mouseleave="onMouseleaveMain">
+        <div class="o-discuss-Call user-select-none d-flex" t-att-class="{'o-fullscreen fixed-top vw-100 vh-100': state.isFullscreen, 'o-minimized': minimized and !env.inChatWindow, 'o-collapsed': state.collapsed, 'position-relative': !state.isFullscreen }">
```

Comment:
You can improve readability by moving `t-att-class` to the next line and placing each class property on its own line, as there are multiple conditions involved.